test/distributed/org/apache/cassandra/distributed/impl/AbstractCluster.java

```diff
@@ -189,12 +189,14 @@
             // those properties may be set for unit-test optimizations; those should not be used when running dtests
             CassandraRelevantProperties.FLUSH_LOCAL_SCHEMA_CHANGES.reset();
             CassandraRelevantProperties.NON_GRACEFUL_SHUTDOWN.reset();
+            CassandraRelevantProperties.IGNORE_MISSING_NATIVE_FILE_HINTS.setBoolean(true);
         }
 
         public AbstractBuilder(Factory<I, C, B> factory)
         {
             super(factory);
             withSharedClasses(SHARED_PREDICATE);
+            withRoot(org.apache.cassandra.io.util.Files.newInMemoryFileSystem().getPath("/cassandra"));
```

Comment:
for now I enable by default just to see how CI works, feel it would be best to make this a flag for tests to opt-in/out of